tools/nlp-toolkit/README.md

```diff
@@ -0,0 +1,40 @@
+# NLP Toolkit
+
+This is a tool to to perform NLP operations using the library [compromise](https://www.npmjs.com/package/compromise).
+
```

Comment:
Maybe we should add a list of the methods we currently support here also

Reply:
Added.

Reply:
Nice one, could be good to use a bullet point list for these later down the line